packages/terra-form-textarea/docs/MigrationGuide.md

```diff
@@ -0,0 +1,19 @@
+# Migrating from terra-form's textarea Component to terra-form-textarea
+
+The following sections describe how to migrate from the textarea component provided in the terra-form package to the textarea component provided in the terra-form-textarea package. Terra-form-textarea provides enhanced API and layout to begin the first phase of the OSC uplift.
+
+## Prop Changes
+
+#### Props Added
+- `disabled` - Whether the textarea is disabled.
+- `isAutoResizable` - indicates whether the textarea can be auto-resized vertically.
+- `isInvalid` - indicates whether or not the textarea is invalid.
+- `size` - the size of the textarea.
+
+## Behavior Changes
+
+#### Added/Changed Behavior
```

Comment:
IMO it makes sense to explain `isAutoreSizable` here, but it feels like the prop description covers the behavior of the other props

Reply:
Removed migration guide. https://github.com/cerner/terra-core/pull/982/commits/802777623b56e1099bdf699936a4de4e2853d6ff